docs/cellxgene_census_docsite_data_release_info.md

```diff
@@ -1,23 +1,110 @@
+### Long-term supported (LTS) Census releases
 
 To enable data stability and scientific reproducibility, [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) plans to perform regular LTS Census data releases:
 
 * Published online every six months for public access, starting on May 15, 2023.
 * Available for public access for at least 5 years upon publication.
  
+The latest LTS Census data release is the default opened by the APIs and recognized as the `census_version = "stable"`.
+
+Python
+
+```python
+import cellxgene_census
+census = cellxgene_census.open_soma()
+```
+
+R
+
+```r
+library("cellxgene.census")
+census <- open_soma()
+```
 
 ### Weekly Census release (latest)
 
 [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) ingests a handful of new datasets every week. To quickly enable access to these new data via the Census, [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) plans to perform weekly Census data releases:
 
-* Published online every Monday for public access, except for CZI holidays. 
-* Available for public access for 1 week or until the next latest release is performed, whichever is the longest.
\ No newline at end of file
+* Available for public access for 1 week or until the next latest release is performed, whichever is the longest.
+
+The weekly release can be opened by the APIs by specifying `census_version = "latest"`.
+
+Python
+
+```python
+import cellxgene_census
+census = cellxgene_census.open_soma(census_version = "latest")
+```
+
+R
+
+```r
+library("cellxgene.census")
+census <- open_soma(census_version = "latest")
+```
+
+## List of LTS Census data releases
+
+### LTS 2023-05-15
+
+Open this data release by specifying `census_version = "2023-06-28"` in future calls to `open_soma()`.
+
+#### 🔴 Erratum 🔴  
+
+There are 243,569 number of cells labelled as `is_primary_data = True` for which a fraction of them the label is incorrect. 
+
+Such label indicates that a cell is the primary representation of an observation, otherwise a cell is deemed to be a duplicate representation. Based on their count vectors, these 243,569 number of cells are represented at least twice with `is_primary_data = True`.
+
+This issue will be corrected in the following LTS data release, by identifying and marking only one cell out of the duplicates as  `is_primary_data = True`.
+
+If you wish to use this data release, you can consider filtering out all of these 243,569 cells by using the `soma_joinids` provided in this file [duplicate_cells_census_LTS_2023-05-15.csv.zip](https://github.com/chanzuckerberg/cellxgene-census/raw/773edab79bbdc78eccb26ec4f8211a9b4c98a71a/tools/cell_dup_check/duplicate_cells_census_LTS_2023-05-15.csv.zip). You can filter specific cells by using the `value_filter` or `obs_value_filter` of the querying API functions, for more information follow this [tutorial](https://chanzuckerberg.github.io/cellxgene-census/notebooks/api_demo/census_query_extract.html). 
```

Comment:
Keep in mind that this file lists _all_ of the dups, so the net effect of the filter is to remove all copies of these cells.  Do you want a file that arbitrarily selects one to be `True` and lists only the others?

Reply:
I'm going to go with all of nothing here.